keras/src/backend/openvino/numpy.py

```diff
@@ -910,9 +910,151 @@ def less_equal(x1, x2):
 def linspace(
     start, stop, num=50, endpoint=True, retstep=False, dtype=None, axis=0
 ):
-    raise NotImplementedError(
-        "`linspace` is not supported with openvino backend"
-    )
+    """
+    Return evenly spaced numbers over a specified interval.
+
+    Returns `num` evenly spaced samples, calculated over the
+    interval [`start`, `stop`].
+
+    The endpoint of the interval can optionally be excluded.
+
+    Parameters
```

Comment:
Remove, this doc will never be generated or seen, only the one in `keras.ops`. Also, it's not following the format that we use for docstring.